src/client.rs

```diff
@@ -1802,20 +1763,17 @@ impl Spotify {
     ///
     /// [Reference](https://developer.spotify.com/documentation/web-api/reference/#endpoint-transfer-a-users-playback)
     #[maybe_async]
-    pub async fn transfer_playback<T: Into<Option<bool>>>(
+    pub async fn transfer_playback(
         &self,
         device_id: &str,
-        force_play: T,
+        force_play: Option<bool>,
     ) -> ClientResult<()> {
-        self.endpoint_put(
-            "me/player",
-            &json! ({
-                "device_ids": vec![device_id.to_owned()],
-                "play": force_play.into().unwrap_or(true)
-            }),
-        )
-        .await?;
+        let params = build_json! {
+            "device_ids": [device_id],
+            optional "force_play": force_play,
```

Comment:
The field name should be `play` instead of `force_play`.

Reply:
fixed